RELEASE/scripts/autoscend/auto_mr2019.ash

```diff
@@ -188,6 +188,18 @@ boolean auto_sausageGoblin(location loc, string option)
 	{
 		return false;
 	}
+	
+	// can't equip kramko sausage grinder during certain paths, return false in those paths just like if you didn't own kramko
+	
+	if(my_path() == "Way of the Surprising Fist")
+	{
+		return false;
+	}
+	
+	if(my_path() == "Avatar of Boris")
+	{
+		return false;
+	}
```

Comment:
This would be better as `if (auto_my_path() == "Way of the Surprising Fist" || auto_my_path() == "Avatar of Boris")`

Reply:
I thought auto_my_path() was just for newest paths which are not mafia supported yet?
As for being two different if functions, I was following the codestyle of other functions in that file.

I went ahead and made the change you suggested.

Reply:
I think I am just supposed to hit resolve conversation now that I have applied the changes you suggested? However, it also seems like another issue I was working on got mixed in as well? odd.